lib/wibox/widget/imagebox.lua

```diff
@@ -233,7 +236,12 @@ function imagebox:draw(ctx, cr, width, height)
             cr:clip(self._private.clip_shape(cr, w*aspects.w, h*aspects.h, unpack(self._private.clip_args)))
         end
 
-        cr:scale(aspects.w, aspects.h)
+        if self._private.handle then
+            RsvgRect = Rsvg.Rectangle {x = 0, y = 0, width = w*aspects.w, height = h*aspects.h }
+            svg_surf = cairo.ImageSurface(cairo.Format.ARGB32, w*aspects.w, h*aspects.h)
```

Comment:
the stacktrace from the failed test actually pointing to this line, but anyway you got the idea :)

Reply:
you marked thread as resolved but test is still failing on this line: https://github.com/awesomeWM/awesome/actions/runs/6042238743/job/16477140356?pr=3802